examples/api/cpp/relations.cpp

```diff
@@ -0,0 +1,148 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Mudathir Mohamed, Andres Noetzli
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * A simple demonstration of reasoning about relations with cvc5 via C++ API.
+ */
+
+#include <cvc5/cvc5.h>
+
+#include <iostream>
+
+using namespace cvc5::api;
+
+int main()
+{
+  Solver solver;
+
+  // Set the logic
+  solver.setLogic("ALL");
+
+  // options
+  solver.setOption("produce-models", "true");
+  solver.setOption("finite-model-find", "true");
+  solver.setOption("sets-ext", "true");
```

Comment:
Can you maybe comment on why we need those 2? (And add these comments to .py and .java, too)

Reply:
Done.